js/ui/grid_core/ui.grid_core.columns_view.js

```diff
@@ -152,29 +151,29 @@ exports.ColumnsView = modules.View.inherit(columnStateMixin).inherit({
         return $cell;
     },
 
-    _createRow: function(row) {
-        return $("<tr>")
-            .addClass(ROW_CLASS)
-            .attr("role", "row");
-    },
-
-    _getTableRoleName: function() {
-        return "grid";
+    _createRow: function(rowObject) {
+        var $element = $("<tr>").addClass(ROW_CLASS);
+        if(rowObject) {
```

Comment:
Why is this change needed?

Reply:
Because I moved setting role="grid" to the "grid.core" for the base container and function "_getTableRoleName" moved too.